vou/batch.py

```diff
@@ -30,33 +30,33 @@ def simulate(self, parallel: bool = True):
         generator, allowing for reproducibility.
         """
 
-        def simulate_one_person(self, seed: int):
+        def simulate_one_person(self, param_dict: dict):
```

Comment:
Minor stylistic point - it's not necessary to name variables with their types eg `param_dict` when we have type hints.